### ISSUE TITLE
TypeError: Cannot set property 'handler' of undefined with webpack

### ISSUE THREAD

**Issue by frsechet** [DataDog/serverless-plugin-datadog#24]
## Expected Behavior
running with datadog plugin enabled should behave the same as running without

## Actual Behavior
crash "TypeError: Cannot set property 'handler' of undefined" when running with this plugin and, apparently, webpack

## Steps to Reproduce the Problem

  1. clone this repo: https://github.com/frsechet/datadog-sls-reproduce-issue
  1. npm install
  1. npm start
  1. visit http://localhost:3042
  1. you should see "hello world" but instead you get a crash

This reproduces the issue locally using serverless-offline, but exactly the same thing happens on live AWS environments.

## Specifications

  - Serverless Framework version: 1.60.1
  - Datadog Serverless Plugin version: 0.14.0
  - Lambda function runtime (Python 3.7, Node 10, etc.): node10
  
## Stacktrace
  
  ```
"TypeError: Cannot set property 'handler' of undefined",
"at Module.<anonymous> (/Users/frsechet/clevy/csml/saas/datadog-sls/.webpack/service/datadog_handlers/webpack:/datadog_handlers/myfunc.js:3:1)",
"at Module../datadog_handlers/myfunc.js (/Users/frsechet/clevy/csml/saas/datadog-sls/.webpack/service/datadog_handlers/myfunc.js:109:30)",
"at __webpack_require__ (/Users/frsechet/clevy/csml/saas/datadog-sls/.webpack/service/datadog_handlers/webpack:/webpack/bootstrap:19:1)",
"at /Users/frsechet/clevy/csml/saas/datadog-sls/.webpack/service/datadog_handlers/webpack:/webpack/bootstrap:83:1",
"at Object.<anonymous> (/Users/frsechet/clevy/csml/saas/datadog-sls/.webpack/service/datadog_handlers/myfunc.js:87:10)",
"at Module._compile (internal/modules/cjs/loader.js:776:30)",
"at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)",
"at Module.load (internal/modules/cjs/loader.js:653:32)",
"at tryModuleLoad (internal/modules/cjs/loader.js:593:12)",
"at Function.Module._load (internal/modules/cjs/loader.js:585:3)",
"at Module.require (internal/modules/cjs/loader.js:690:17)",
"at require (internal/modules/cjs/helpers.js:25:18)",
"at Object.createHandler (/Users/frsechet/clevy/csml/saas/datadog-sls/node_modules/serverless-offline/src/functionHelper.js:215:15)",
"at handler (/Users/frsechet/clevy/csml/saas/datadog-sls/node_modules/serverless-offline/src/ApiGateway.js:485:40)",
"at module.exports.internals.Manager.execute (/Users/frsechet/clevy/csml/saas/datadog-sls/node_modules/@hapi/hapi/lib/toolkit.js:41:33)",
"at Object.internals.handler (/Users/frsechet/clevy/csml/saas/datadog-sls/node_modules/@hapi/hapi/lib/handler.js:46:48)",
"at exports.execute (/Users/frsechet/clevy/csml/saas/datadog-sls/node_modules/@hapi/hapi/lib/handler.js:31:36)",
"at Request._lifecycle (/Users/frsechet/clevy/csml/saas/datadog-sls/node_modules/@hapi/hapi/lib/request.js:312:68)"
  ```

**Comment by frsechet** [comment]
note: this might be related to https://github.com/DataDog/serverless-plugin-datadog/issues/10 (closed). For several reasons it would be highly impractical for us to use the manual setup as advised, so just in case, here are some steps to reproduce something similar!

**Comment by DarcyRaynerDD** [comment]
Hi @frsechet, thanks for submitting this issue. I can confirm I can reproduce the error using the repo you provided. I'm investigating the issue now to try and identify a fix.

**Comment by DarcyRaynerDD** [comment]
Hi @frsechet, we just released v0.16.0 of the library which should have a fix for this. Can you confirm whether that addresses the issue?

**Comment by frsechet** [comment]
Looks like it work! I did not have time to upgrade my production stacks yet but the reproduction repo was fixed by the update to 0.16.0 indeed. I don't expect my other stacks to have any further trouble.

Thanks a lot for the swift update!
